src/email_verification/views.py

```diff
@@ -214,25 +184,40 @@ def webhooks(request, topic):
                         "mime-type": "text/plain",
```

Comment:
This is redundant: default null implies text/plain, and specifying it will generate a mime-type record per credential in the wallet where it doesn't need one.